<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="T4_Add_CCF_return_but">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File:</string>
   </property>
  </widget>
  <widget class="QFrame" name="T4_Add_CC_rad_frame">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="T4_Add_CCF_w_e_rad">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>w/e</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="T4_Add_CCF_w_o_rad">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>w/o</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="T4_Add_CCF_n_a_rad">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>n/a</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="T4_Add_CCF_reload_but">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </widget>
  <widget class="QPushButton" name="T4_Add_CCF_execute_but">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enclosure:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="T4_Add_CCF_file_le">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>T4_Add_CCF_file_le</tabstop>
  <tabstop>T4_Add_CCF_w_e_rad</tabstop>
  <tabstop>T4_Add_CCF_w_o_rad</tabstop>
  <tabstop>T4_Add_CCF_n_a_rad</tabstop>
  <tabstop>T4_Add_CCF_execute_but</tabstop>
  <tabstop>T4_Add_CCF_return_but</tabstop>
  <tabstop>T4_Add_CCF_reload_but</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
